core/blockchain.go

```diff
@@ -1193,7 +1204,7 @@ func (bc *BlockChain) InsertReceiptChain(blockChain types.Blocks, receiptChain [
 		// a background routine to re-indexed all indices in [ancients - txlookupLimit, ancients)
 		// range. In this case, all tx indices of newly imported blocks should be
 		// generated.
-		var batch = bc.db.NewBatch()
+		batch := bc.db.NewBatch()
```

Comment:
I don't know, what is this change about. Let's revert to keep as less invasive as possible

Reply:
Hmm, it is my editor :\
I am not sure, how to disable that